builder/ConfTargets.cmake

```diff
@@ -76,6 +76,7 @@ option( MFX_ENABLE_USER_DECODE "Enabled user decode plugins?" ON)
 option( MFX_ENABLE_USER_ENCODE "Enabled user encode plugins?" ON)
 option( MFX_ENABLE_USER_ENC "Enabled user ENC plugins?" ON)
 option( MFX_ENABLE_USER_VPP "Enabled user VPP plugins?" ON)
+option( MFX_ENABLE_VP9_VIDEO_DECODE "Enabled VP9 decoder?" ON)
```

Comment:
That's wrong. And means that you don't check what you have done. Options which you place _after_ include(${MFX_CONFIG_FILE}) won't take effect! I will submit patch to fix that for the bunch of USER options above.